src/main/resources/config.yml

```diff
@@ -119,6 +119,12 @@ hearts-on-death: true
 # If true, player will die directly when get damage from void
 die-on-void: false
 
+# Ressourcespawner spawns ressourcen inside of chests if it's located on spawner position
```

Comment:
This description is kinda duplicating the description below "type: boolean"
I don't think we need it here.